tools/coverage/README.md

```diff
@@ -14,6 +14,16 @@ The code coverage tool has two major features:
 
 See the [design document](design.md).
 
+## Skipped Files
+
+This tool allows users to specify which files to skip, from reporting and aggregation.
+User can add the git attribute `coverage-excluded=true` to any directory or file to exclude files.
```

Comment:
leave off the =true
(yes, it can be used that way, but git attributes already can be a binary value so why make it a string value of "true"?)

Reply:
Good to know.
However, please take a look at the example linked. It's to be consistent with the rest of the file.